packages/tinacms/src/tina-cms.ts

```diff
@@ -106,4 +106,16 @@ export class SidebarState extends Subscribable {
     this._isOpen = nextValue
     this.notifiySubscribers()
   }
+
+  private _isHidden: boolean = this.hidden
+
+  get isHidden() {
```

Comment:
Looks good! `SidebarState` already has the `hidden` flag. If this was switched to `hidden` then it would be a non-breaking change.

Reply:
Good call, [this commit](https://github.com/tinacms/tinacms/pull/769/commits/12f71620bd878d814069f2e9a6bea20bf4a087b0) should do that.